back/test/integration/TestTechMapsApi.js

```diff
@@ -63,26 +62,331 @@ describe("API", () => {
       expect(res).to.have.status(404);
     });
 
-    it.skip("should bump up version when modified techmap put", async() => {
+    it("should bump up version when modified techmap put", async () => {
       // todo: make sure all fields are checked.
+      const id = newTechMapId();
+
+      const newTechMap = {
+        id,
+        name: "Хліб Французький (КХ)",
+        units: [1, 6],
+        steps: [
+          {
+            id: "STP-XXX-YYY-ZZZ-YY0",
```

Comment:
This will result in 400 bad request since we are going to have validation (BTW: this needs to be validated in separate tests)

Reply:
Do you mean we need a function similar to newTechMapId() so that id contained a valid uuid?